torch/testing/_internal/common_nn.py

```diff
@@ -1849,248 +2065,288 @@ def fractional_max_pool3d_test(test_case):
     dict(
         module_name='PixelShuffle',
         constructor_args=(3,),
+        cpp_constructor_args='torch::nn::PixelShuffleOptions(3)',
         input_size=(1, 9, 4, 4),
     ),
     dict(
         constructor=wrap_functional(F.interpolate, size=12, scale_factor=None, mode='nearest'),
+        cpp_options_args='F::InterpolateFuncOptions().size(std::vector<int64_t>({12})).scale_factor(c10::nullopt).mode(torch::kNearest)',
```

Comment:
just for my understanding: What happens in this case, where we have `cpp_options_args` but no `cpp_function_call`? What does the `F::interpolate` function call look like, do we pass the options directly to it?

Reply:
That's right - in the case where we have `cpp_options_args` but no `cpp_function_call`, the C++ function call would look like: `F::interpolate(input, cpp_options_args)`.

In general, if we only have `cpp_options_args`, the function call is `F::functional_name(input, target, extra_args_1, extra_args_2, ..., cpp_options_args)`

Reply:
I added an assert to check that there has to be one and only one of `cpp_options_args` and `cpp_function_call` that exists in the test dict :D